Rogue-Ape-Studios/Crusader
# ✨ Critical hits

## Content

Created a really simple way for the player to occasionally deal critical damage.

## Changes


### Updated
`Assets/Prefabs/Player/Player_1.prefab` : Setup the ability to crit.
`Assets/Scripts/Player/Combat/AttackDamage.cs` : Implemented the ability to crit.

## Testing

The feature can be testing by following these steps:

1. Check to see if critical hits are enabled.
2. Start the scene.
3. You have a 20% chance to crit.


Closes #193
